### PR TITLE
GH1173: Adds integration tests for Cake.Common.Tools.DotNetCore.DotNetCoreAliases

### PR DESCRIPTION
**Pull request by devlead**
* releates to #1173

### CODE REVIEW

File: tests/integration/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cake [hunk 1]
```diff
@@ -0,0 +1,153 @@
+#load "./../../../utilities/xunit.cake"
+#load "./../../../utilities/paths.cake"
+
+// todo copy to temp
```
Comment (gep13) [inline tests/integration/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cake]: @devlead why is this a `todo`?

Comment (devlead) [inline tests/integration/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cake]: it's not. I'll remove.

File: tests/integration/resources/Cake.Common/Tools/DotNetCore/hwapp.tests/Tests.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+namespace HWApp.Tests
+{
+    public sealed class GreeterTests
+    {
+        public sealed class TheGreaterMethod
+        {
+            [Fact]
+            public void Should_Not_Fail_Test()
+            {
+                Assert.NotEqual("true", Environment.GetEnvironmentVariable("hwapp_fail_test"));
```
Comment (gep13) [inline tests/integration/resources/Cake.Common/Tools/DotNetCore/hwapp.tests/Tests.cs]: @devlead I see what you did here! Very cool :-)

File: tests/integration/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cake (hunk 1 above)
Comment (devlead) [inline tests/integration/Cake.Common/Tools/DotNetCore/DotNetCoreAliases.cake]: Removed.

**Pull request merged**